src/MarketingPageAcceptanceTests.Actions/Pages/ContactDetails.cs

```diff
@@ -16,27 +16,45 @@ public bool PageDisplayed()
             return new Common(driver).PageTitleEquals("Contact details");
         }
 
-        public void EnterAllData(IContactDetail firstContact, IContactDetail secondContact = null)
+        public void EnterAllData(IContactDetail firstContact, IContactDetail secondContact = null, bool clearFirst = false)
         {
-            FirstContactComplete(firstContact);
+            FirstContactComplete(firstContact, clearFirst);
 
             if(!(secondContact is null))
             {
-                SecondContactComplete(secondContact);
+                SecondContactComplete(secondContact, clearFirst);
             }
         }
 
-        private void FirstContactComplete(IContactDetail contact)
+        private void FirstContactComplete(IContactDetail contact, bool clearFirst = false)
         {
+            if (clearFirst)
+            {
+                driver.FindElement(pages.ContactDetails.Contact1FirstName).Clear();
+                driver.FindElement(pages.ContactDetails.Contact1LastName).Clear();
+                driver.FindElement(pages.ContactDetails.Contact1EmailAddress).Clear();
+                driver.FindElement(pages.ContactDetails.Contact1PhoneNumber).Clear();
+                driver.FindElement(pages.ContactDetails.Contact1JobSector).Clear();
+            }
+
             driver.FindElement(pages.ContactDetails.Contact1FirstName).SendKeys(contact.FirstName);
             driver.FindElement(pages.ContactDetails.Contact1LastName).SendKeys(contact.LastName);
             driver.FindElement(pages.ContactDetails.Contact1EmailAddress).SendKeys(contact.EmailAddress);
             driver.FindElement(pages.ContactDetails.Contact1PhoneNumber).SendKeys(contact.PhoneNumber);
             driver.FindElement(pages.ContactDetails.Contact1JobSector).SendKeys(contact.JobSector);
         }
 
-        private void SecondContactComplete(IContactDetail contact)
+        private void SecondContactComplete(IContactDetail contact, bool clearFirst = false)
```

Comment:
You're directly implementing this in the calling method, do you need to set a default value here?